sequencing-server/src/routes/command-expansion.ts

```diff
@@ -49,6 +49,7 @@ commandExpansionRouter.post('/put-expansion', async (req, res, next) => {
     return next();
   }
 
+  // WHY NOT DO THIS FIRST?
```

Comment:
@pranav-super do you feel like this comment has been addressed?

Reply:
I don't feel like it has been, however as it's not the focus of the PR this comment is fine safe to remove.

Reply:
This inefficiency is referenced [here](https://github.com/NASA-AMMOS/aerie-team/discussions/57#discussioncomment-12554051).